internal/apiclient/pricing.go

```diff
@@ -55,8 +55,17 @@ func NewPricingAPIClient(ctx *config.RunContext) *PricingAPIClient {
 		}
 	}
 
+	skipVerify := false
+	if ctx.Config.TLSInsecureSkipVerify != nil {
+		skipVerify = *ctx.Config.TLSInsecureSkipVerify
+	} else if len(rootCAs.Subjects()) == 0 && ctx.Config.TLSCACertFile == "" {
```

Comment:
In this case, should we just pass a nil `tlsConfig` to the API client instead of skipping verify? If we do that will it still work with Windows and verify the pricing.api.infracost.io cert?